cmd/kubectl-gadget/advise/seccomp-policy.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 var seccompAdvisorCmd = &cobra.Command{
-	Use:   "seccomp-advisor",
+	Use:   "seccomp-policy",
 	Short: "Generate seccomp policies based on recorded syscalls activity",
```

Comment:
Same here, it should say profiles rather than policies

Reply:
Done.